src/gui/MainWindow.cpp

```diff
@@ -530,6 +530,12 @@ void MainWindow::updateWindowTitle()
         windowTitle = QString("%1 - %2").arg(customWindowTitlePart, BaseWindowTitle);
     }
 
+    if (customWindowTitlePart.isEmpty() || stackedWidgetIndex == 1) {
+        setWindowFilePath("");
```

Comment:
@phoerious The default value is an empty string but it will remember the last value set by `setWindowFilePath()` so it will add the the proxy icon in some unwanted situations such as the settings window which has a different window title without the file name.